packages/web3-constants/evm/ito.json

```diff
@@ -58,7 +64,9 @@
         "Fantom": "0x981be454a930479d92C91a0092D204b64845A5D6",
         "Aurora": "0x2cf91AD8C175305EBe6970Bd8f81231585EFbd77",
         "Aurora_Testnet": "0xdcA6F476EebCDE8FE8b072e3fC80dBC28dC209b3",
-        "Conflux": "0x066804d9123bf2609ed4a4a40b1177a9c5a9ed51"
+        "Conflux": "0x066804d9123bf2609ed4a4a40b1177a9c5a9ed51",
+        "Harmony": "0x5b966f3a32db9c180843bcb40267a66b73e4f022",
```

Comment:
Can confirm ITO contract for harmony is 0x5b966f3a32db9c180843bcb40267a66b73e4f022

Reply:
Yes, ref: https://github.com/DimensionDev/InitialTwitterOffering#contract-address

Reply:
> Yes, ref: [DimensionDev/InitialTwitterOffering#contract-address](https://github.com/DimensionDev/InitialTwitterOffering#contract-address)

Hi, @developerfred, thanks. I have confirmed it, and just leave a memo here so that I don't need to check it again.

You don't need to add a reference.
And you should keep this conversation unresolved.

Reply:
ok